<?xml version="1.0"?>

<robot name="mkz" xmlns:xacro="http://www.ros.org/wiki/xacro" >

  <!-- Necessary includes -->
  <xacro:include filename="$(find dbw_mkz_description)/urdf/vehicle_structure.urdf.xacro" />
  <xacro:include filename="$(find dbw_mkz_description)/urdf/vehicle_gazebo.urdf.xacro" />

  <!-- Bolt-on sensors go here -->
  <!--<xacro:include filename="$(find dbw_mkz_description)/urdf/vehicle_sensors.urdf.xacro" />-->
  <xacro:include filename="$(find vero_sim)/urdf/hokuyo.urdf.xacro" />

  <!--<xacro:dbw_mkz_camera name="front_camera" parent_link="base_footprint" x="2.0" y="0" z="1.3" roll="0" pitch="0.6" yaw="0" />-->
  <!--<xacro:dbw_mkz_gps name="perfect_gps" parent_link="base_footprint" x="0.6" y="0" z="1.45" rate="50.0" ref_lat="45.0" ref_lon="-81.0" />-->
  <xacro:sensor_hokuyo name="hokuyo" parent_link="base_footprint" x="3.90" y="0.0" z="0.6" roll="0.0" pitch="0.0" yaw="0.0" />

</robot>
